kong/runloop/handler.lua

```diff
@@ -18,6 +18,7 @@ local constants   = require "kong.constants"
 local singletons  = require "kong.singletons"
 local certificate = require "kong.runloop.certificate"
 local concurrency = require "kong.concurrency"
+local regex       = require "ngx.re"
```

Comment:
sorry about a "style" request, but could you name this `ngx_re` (or alternative localize the function as `re_split = ngx.re.split`) to match how ngx.re is localized in other parts of the code? I was reading the other part of the patch and for a moment I thought `regex` was something else.

Reply:
Changed that.